qa/tasks/mgr/mgr_test_case.py

```diff
@@ -89,6 +92,9 @@ def setup_mgrs(cls):
             cls.mgr_cluster.mon_manager.raw_cluster_cmd(
                 "mgr", "module", "disable", m)
 
+        cls.mgr_cluster.mon_manager.raw_cluster_cmd('config', 'set', 'mon', 'mon_allow_pool_delete', 'true')
+        cls.mgr_cluster.mon_manager.raw_cluster_cmd('osd', 'pool', 'rm', '.mgr', '.mgr', '--yes-i-really-really-mean-it-not-faking')
```

Comment:
I was wondering if there are any scenarios where the .mgr pool needs to persist between tests; I guess not, but just a thought

Reply:
It's a valid concern but I think in this case it's appropriate for testing the mgr modules.